pyproject.toml

```diff
@@ -81,3 +83,27 @@ include = [
 exclude = [
   ".*",
 ]
+
+[tool.mypy]
+python_version = "3.8"
```

Comment:
```suggestion
python_version = "3.10"
```

Reply:
This needs to the earliest version of Python that you intend to support.

Reply:
I suggest reverting this